paasta_tools/cli/cmds/spark_run.py

```diff
@@ -482,6 +482,20 @@ def get_spark_config(
             # Update default configuration
             user_args[fields[0]] = fields[1]
 
+    if 'spark.sql.shuffle.partitions' not in user_args:
+        user_args['spark.sql.shuffle.partitions'] = str(
+            2 * int(user_args['spark.cores.max']),
+        )
+        paasta_print(
```

Comment:
Why do we need to print a warning?
Why can't this just be a "sane default" for users?

Reply:
1. I believe there is value in warning that we are setting a default for this parameter that is not the spark default. 
2. Too many partitions can cause a sizeable overhead e.g., a job with 800 cores can emit 1600 partitions!! The default value that we're setting is a better guess than what spark provides but may still be bad. 
3. There is quite a bit of information setting the right value of spark.sql.shuffle.partitions, depending on the amount of data as well; something that we're not doing.